baytune/tuning/metamodels/gaussian_process.py

```diff
@@ -96,5 +98,7 @@ def _fit(self, trials, scores):
     def _predict(self, candidates):
         trans_candidates = self._transform(candidates)
         predicted = super()._predict(trans_candidates)
-
-        return self._score_distribution.ppf(scipy.stats.norm.cdf(predicted))
+        cdf = scipy.stats.norm.cdf(predicted)
+        if len(cdf.shape) == 2:
```

Comment:
I am not 100% sure on this. This could be wrong but was needed for the copulas upgrade

Reply:
The change makes sense, change is required when Copulas changed from [Before](https://github.com/sdv-dev/Copulas/blob/5e6bf28ad9a1d05b52c0653dd7393e36269b9ade/copulas/univariate/gaussian_kde.py) to [After](https://github.com/sdv-dev/Copulas/blob/e70f6896330837b5a17be6bae7714f41245d241d/copulas/univariate/gaussian_kde.py)

`self._score_distribution.ppf` directly calls `self.percent_point` which contained the following before it was removed in the change:
`if len(U.shape) == 2:
    return np.fromiter(
        (self.percent_point(u[0]) for u in U),
        np.dtype('float64')
    )`
After the change, it just throws an error:
`if len(U.shape) > 1:
    raise ValueError("Expected 1d array, got %s." % (U, ))`